arci/src/clients/dummy_transform_resolver.rs

```diff
@@ -0,0 +1,38 @@
+use nalgebra::{Isometry3, Vector3};
+
+use crate::{Error, TransformResolver};
+
+/// Dummy TransformResolver for debug or tests.
+#[derive(Debug)]
+pub struct DummyTransformResolver {
+    pub transformation: Isometry3<f64>,
+}
+
+impl DummyTransformResolver {
+    /// Creates a new `DummyTransformResolver`.
+    pub fn new() -> Self {
+        Self {
+            transformation: Isometry3::new(
+                Vector3::new(0.0, 0.0, 0.0),
+                Vector3::new(0.0, 0.0, 0.0),
+            ),
```

Comment:
Please use identity() instead.

```suggestion
            transformation: Isometry3::identity(),
```